packages/core/src/middleware/koa-connector-error-handler.ts

```diff
@@ -21,77 +21,32 @@ export default function koaConnectorErrorHandler<StateT, ContextT>(): Middleware
       const errorMessage = conditional(result.success && '\n' + result.data.errorDescription);
```

Comment:
```suggestion
      const errorMessage = conditional(result.success && '\n' + result.data.errorDescription);
      const code = `connector.${code}`;
```

then 

```ts
throw new RequestError({ code, status: 400 }, data);
```

Reply:
This is not viable since we need to check the equality between `code` and corresponding `ConnectorErrorCodes` string literals.